install.sh

```diff
@@ -59,7 +59,7 @@ make_virtualenv() {
     echo "Making Virtual Environment for [$1] in $VENV_DIR"
     python3 -m venv $VENV_DIR/$1
     source $VENV_DIR/$1/bin/activate
-    python3 -m pip install --upgrade pip setuptools
+    python3 -m pip install --upgrade pip setuptools wheel
```

Comment:
not necessarily related to the topic in the PR, but ok

Reply:
True lol.
I don't want to make a separate PR for a small change, esp given how long it take for the CI to run.